go/pkg/sysdb/coordinator/coordinator.go

```diff
@@ -239,8 +239,8 @@ func (s *Coordinator) ListCollectionsToGc(ctx context.Context, cutoffTimeSecs *u
 	return s.catalog.ListCollectionsToGc(ctx, cutoffTimeSecs, limit)
 }
 
-func (s *Coordinator) ListCollectionVersions(ctx context.Context, collectionID types.UniqueID, tenantID string, maxCount *int64, versionsBefore *int64, versionsAtOrAfter *int64) ([]*coordinatorpb.CollectionVersionInfo, error) {
-	return s.catalog.ListCollectionVersions(ctx, collectionID, tenantID, maxCount, versionsBefore, versionsAtOrAfter)
+func (s *Coordinator) ListCollectionVersions(ctx context.Context, collectionID types.UniqueID, tenantID string, maxCount *int64, versionsBefore *int64, versionsAtOrAfter *int64, includeMarkedForDeletion bool) ([]*coordinatorpb.CollectionVersionInfo, error) {
```

Comment:
Mind giving the context on this change for my knowledge?

Reply:
this method would previously always filter out versions marked for deletion
this allows listing versions marked for deletion so we can make assertions during tests